### ISSUE TITLE
Generate accessibility report for the website

### ISSUE THREAD

**Issue by s-sajid** [BThoom/ability-workshop-calgary#12]

**Comment by scribblemaniac** [comment]
Here are Lighthouse accessiblity reports for all the current pages on the website:
[lighthouse-reports.zip](https://github.com/BThoom/ability-workshop-calgary/files/11857885/lighthouse-reports.zip)

The following are issues identified by the reports:
- [x] Add title to iframe on donation page
- [x] Add title to projects page
- [x] Sequentially order header tags in projects galleries (gallery titles are h3 but card titles are h5)
- [ ] <del>Improve contrast ratio on contact form submit button (change bootstrap theme?) #9</del> (button removed)

Additionally the following things should be checked manually on all pages:
- [x] The page has a logical tab order
- [x] Interactive controls are keyboard focusable
- [x] Interactive elements indicate their purpose and state
- [x] The user's focus is directed to new content added to the page
- [x] User focus is not accidentally trapped in a region
- [ ] Custom controls have associated labels
- [x] Custom controls have ARIA roles
- [x] Visual order on the page follows DOM order
- [ ] Offscreen content is hidden from assistive technology
- [x] HTML5 landmark elements are used to improve navigation

**Comment by scribblemaniac** [comment]
> Custom controls have associated labels

The next and previous buttons for the carousel have hidden span in the buttons for screen readers, but it may make more sense to use hidden labels. This may be a good thing to ask Inclucity.

> Offscreen content is hidden from assistive technology

The is a big issue with the current donation page, and there's not much we can do about it while we are tied to the Shaw Charity Classic.
